### ISSUE TITLE
Telegram, Slack, Forums, IRC

### ISSUE THREAD

**Issue by Zverik** [EdwardBetts/localosm#2]
There are other channels where OSMers live. In Russia, for example, the mailing list is empty, but the majority of mappers live on the forum and in @ruosm telegram channel. In US, I hear, many people live in Slack. And there are also regional IRC channels on the OFTC server.

**Comment by EdwardBetts** [comment]
Yes, this makes sense. I'm going to extend the data model to support channels other than mailing lists on lists.openstreetmap.org.

**Comment by Zverik** [comment]
Thanks! Some links for reference:

* https://wiki.openstreetmap.org/wiki/IRC#Other_chat_rooms
* https://wiki.openstreetmap.org/wiki/List_of_OSM_centric_Telegram_accounts
* https://forum.openstreetmap.org/index.php#idx4
* https://wiki.openstreetmap.org/wiki/Slack

**Comment by andrewharvey** [comment]
there seems to be a lot of overlap between this project and https://github.com/osmlab/osm-community-index, but the latter seems to support non mailing list channels

**Comment by EdwardBetts** [comment]
Thanks @andrewharvey, I'd hadn't seen https://github.com/osmlab/osm-community-index
